src/client/components/DataGrid/DataCell/DataCell.scss

```diff
@@ -134,9 +133,27 @@ $border: 1px solid $ui-border;
     &.lastCol {
       border-left: $border;
     }
+
+    &.highlighted {
+      &.firstCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-right, $highlight-border-shadow;
+        clip-path: inset(-4px -4px -4px 0px); // Adds shadow to top, bottom, right
+      }
+
+      &.lastCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-left, $highlight-border-shadow;
+        clip-path: inset(-4px 0px -4px -4px); // Adds shadow to top, bottom, left
+      }
+
+      &.firstCol.lastCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-left, $highlight-right, $highlight-border-shadow;
+        clip-path: inset(-4px -4px -4px -4px); // Adds shadow to top, bottom, left, right
+      }
+    }
   }
 
-  .data-cell.editable {
+  .data-cell.editable:not(.noBorder) {
```

Comment:
This is to fix this border by the review indicator, which is added unnecessarily:
![image](https://github.com/openforis/fra-platform/assets/41337901/3c92b1ca-7086-48e5-8745-2f8d7767cee7)

Reply:
Yes, since this component is in a DataRow, the DataCell got the lastCol prop and the styles in the `rtl` class were not updated accordingly (overlooked by me, it passed this time me that Arabic has some different styles 😅). Will consider that when testing from now on. 

I've tested now for Arabic too, but I'll double check 👍

Reply:
Tested in Arabic and other languages UI styling and functionality in the mentioned sections today, and it works as expected. I couldn't think of any other special cases. If you think there might be something else please let me know, otherwise I think this is good to go 👍